### ISSUE TITLE
Accesing phi and res attributes of PEST control file with cov mat regularisation

### ISSUE THREAD

**Issue by rhugman** [pypest/pyemu#352]
Hi,

When reading a PEST(_HP) control file that has covariance matrix file assigned for regularisation of a "regul..." observation group, and then accessing the Pst.phi or Pst.res attributes returns the error: 
`ParserError: Error tokenizing data. C error: Expected 11 fields in line 11675, saw 12`

I assume this is because PEST_HP writes `Cov. Mat.` to the *.res file , which then gets parsed as 2 columns?:

```
'Name                 Group          Measured         Modelled         Residual         Weight       Weight*Measured    Weight*Modelled  Weight*Residual  Measurement_sd   Natural_weight
kx1:1                regulkx1      0.3010300        0.3010300        4.3360196E-09     Cov. Mat.           na               na               na               na               na
```

**Comment by jtwhite79** [comment]
That looks to be the case.  Its hard to parse whitespace delim files when where are extra whitespaces unless they are quoted...not sure what we can do in this case...maybe a fnx to clean up a res file?  Maybe pandas has some ways to help with this?

**Comment by rhugman** [comment]
Specifying `usecols=header` to  line 255 in `pst_utils.py` gets around it. 

```
    res_df = pd.read_csv(
        f, header=None, names=header, sep=r"\s+", converters=converters, usecols=header )

```

**Comment by rhugman** [comment]
Should be OK; I think it will only lose the “extra” column caused by the space in Cov. Mat.. That column is full of NaNs anyway…I’ll test and submit a Pr tmrw.
